stablehlo/tests/stablehlo_legalize_to_vhlo.0_16_0.mlir

```diff
@@ -0,0 +1,2290 @@
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo --mlir-print-op-generic --split-input-file %s | FileCheck %s
+// RUN: stablehlo-translate --serialize --target=current %s | stablehlo-translate --deserialize | stablehlo-opt > %t.0
+// RUN: stablehlo-opt %s > %t.1
+// RUN: diff %t.0 %t.1
+// RUN: stablehlo-translate --serialize --target=current %s | stablehlo-opt --pass-pipeline='builtin.module(stablehlo-deserialize)' > %t.0
```

Comment:
Also looks like the bytecode file is empty. Can you generate it using the command from:
https://github.com/openxla/stablehlo/blob/main/docs/vhlo.md#add-versioned-serialization-test

Note the `strip-debuginfo` flag is important since it removes any path-dependent info.

Reply:
Done. Forgot to run the last command originally 🙃